ale/drivers/mro_drivers.py

```diff
@@ -109,13 +127,22 @@ def instrument_id(self):
         : str
           instrument id
         """
-        return self.id_lookup[self.label['INSTRUMENT_NAME']]
+        id_lookup = {
```

Comment:
I went ahead and threw this straight into the only property that makes use of it. This should be fine so long as no one else needs to access this value for whatever reason. 

I did also not include it in the preceding property as it did not make use of it. I hope this was ok as well, but someone check my logic.